README.md

```diff
@@ -31,6 +31,11 @@ An `$ExpectType` directive tests the type of the expression on the next line. Th
     npm install -g typings-checker
     typings-checker your-test.ts
 
+## options
+
+* `--noLines` (`-l`): hide line numbers in output -- useful if you don't want these clogging diff logs when you add or remove a line.
```

Comment:
My inclination would be to just leave both of these options out. If you want to filter out line numbers, you can do it with a bash one-liner:

    typings-checker test.ts | perl -pe 's/\.ts:(\d+)/.ts/'

But I still don't totally understand the use case. Wouldn't you always want zero assertion failures, just like you wouldn't want to commit code that doesn't compile?